### ISSUE TITLE
Definition list splitting

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rst2pdf/rst2pdf#286]
```
Definition lists should **never** split between the term and the definition.
```

Original issue reported on code.google.com by `roberto.alsina` on 17 Mar 2010 at 12:21

**Comment by lornajane** [comment]
Can we identify which tests these were and update the issue?

**Comment by ralsina** [comment]
The test is in rst2pdf/tests/input/test_issue_286.txt

**Comment by ralsina** [comment]
This is still broken. in test_issue_286.pdf in the first page it should not break after "Term:" and it should move it to the next page to keep it together with the definition.

Marking the current checksum as bad and adding an ignore file

**Comment by CendioOssman** [comment]
This seems to have been changed in 28a5f0c620346c52fa41e574a7395524369065e7. However it is not mentioned in the commit message, so perhaps the change was accidental?

**Comment by akrabat** [comment]
> This seems to have been changed in [28a5f0c](https://github.com/rst2pdf/rst2pdf/commit/28a5f0c620346c52fa41e574a7395524369065e7). However it is not mentioned in the commit message, so perhaps the change was accidental?

Yes. I imagine it was an unintended side-effect of ensuring that a list can go over multiple pages.

**Comment by CendioOssman** [comment]
Is that still an issue though? I assume that lists using `SplitTable` means that it can be split over multiple pages? So setting `keepWithNext` on the preceding paragraph should just mean that the first chunk of the table and the paragraph are kept together?

**Comment by akrabat** [comment]
I imagine it needs playing with to see which, if any, tests break.

ReportLab now has `ListFlowable()` & `ListItem(), so it's possible that we may want to rejig everything related to lists…

**Comment by ralsina** [comment]
Can I be unassigned? I am no longer active.
